remote_exec/remote_exec.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+option go_package = "teleport_exec/remote_exec";
+package remote_exec;
+
+//-----------------------------------------------------------------------------
+message StartCommandRequest { string command = 1; }
```

Comment:
Yes, the command it expected to be a fully-formed shell command string ready to be passed into a `bash -c` as a single argument.

Reply:
You are planning to execute through bash? Why not just directly execute processes?

Reply:
TBH, it just felt easier to accept a fully-formed command and feed it to a shell instead of having to deal with arguments, escaping, etc. Additionally, it makes commands much easier to reason about from a client's point of view (one could do things like "FOO=bar some_command" and it would just work.

Alternatively, I could accept an array of strings and feel those into `exec.Command` and people who need environment variables could so the shell -c trick themselves (or we could add an option to pass environment variables in like Docker does it).

Reply:
Taking an array of strings and feeding into `exec.Command` sounds great.